lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -42,7 +42,14 @@
 #include <maya/MUintArray.h>
 #include <maya/MViewport2Renderer.h>
 
+#if PXR_VERSION > 2008
+#else
```

Comment:
It is better to remove the empty conditional compilation branch whose PXR_VERSION is larger than 2008.

Reply:
The reason I wrote it this way was to be identical to the ifdef used in the _LoadImage() function, to avoid subtle issue if I were to inverse the logic. (for example comparing to <= 2008 instead.) But I can change it.